docs/lib/reference/stores/memory.rst

```diff
@@ -0,0 +1,5 @@
+memory
```

Comment:
Why do we use small letters here and not title case? I noticed the references r like this

Reply:
Because these names always reflect the module name verbatim